### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -17,48 +17,42 @@
...
+    filesMatching('agent.properties') {
+        expand(project.properties)
+    }
 }
 
 dependencies {
     if (releaseMode) {
         compile 'com.epam.reportportal:client-java:5.0.0-BETA-4'
-        compile 'com.epam.reportportal:commons-model:5.0.0-BETA-11'
+        compile 'com.epam.reportportal:commons-model:5.0.0-RC-6'
```
Comment (HardNorth) [inline build.gradle]: Please update with 5.0.0

File: build.gradle [hunk 2]
```diff
@@ -17,48 +17,42 @@
...
+processResources {
+    project.properties.ext.set('agent', "$project.name:$project.version")
+    filesMatching('agent.properties') {
+        expand(project.properties)
+    }
 }
 
 dependencies {
     if (releaseMode) {
         compile 'com.epam.reportportal:client-java:5.0.0-BETA-4'
```
Comment (HardNorth) [inline build.gradle]: Please update with client-java:5.0.0-BETA-7

File: src/main/resources/agent.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+agent=${agent}
```
Comment (HardNorth) [inline src/main/resources/agent.properties]: Please add newline

File: src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+	private static ItemAttributesRQ osInfo() {
+		String value =
+				System.getProperty("os.name") + SEPARATOR + System.getProperty("os.arch") + SEPARATOR + System.getProperty("os.version");
+		return new ItemAttributesRQ(OS_KEY, value, true);
+	}
+
+	private static ItemAttributesRQ agentInfo() {
+		String agent = null;
+		try (InputStream inputStream = new FileInputStream("agent.properties")) {
```
Comment (HardNorth) [inline src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java]: Please use Classloader.getResourceAsStream()

File: src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+import org.slf4j.LoggerFactory;
+import rp.com.google.common.collect.Sets;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.Set;
+
+public class SystemAttributesFetcher {
```
Comment (HardNorth) [inline src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java]: Please add javadoc.
And unit-tests for this class.

File: build.gradle (hunk 1 above)
Comment (ihar-kahadouski) [reply build.gradle]: done

File: build.gradle (hunk 2 above)
Comment (ihar-kahadouski) [reply build.gradle]: done

File: src/main/resources/agent.properties (hunk 3 above)
Comment (ihar-kahadouski) [reply src/main/resources/agent.properties]: done

File: src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java (hunk 4 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java]: done

File: src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java (hunk 5 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java]: there are no public methods

Comment (HardNorth) [reply src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java]: Doesn't mean that there is no bugs.
Also you call `static Set<ItemAttributesRQ> collectSystemAttributes(Boolean skippedAnIssue)` which is package-public and allows you to implement a test in the same package.

Comment (ihar-kahadouski) [reply src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java]: done

**Pull request merged**